pkg/eventingtls/eventingtls.go

```diff
@@ -45,6 +45,14 @@ const (
 	DefaultMinTLSVersion = tls.VersionTLS12
 	// SecretCACrt is the name of the CA Cert in the secret
 	SecretCACert = "ca.crt"
+	// IMCDispatcherServerTLSSecretName is the name of the tls secret for the imc dispatcher server
+	IMCDispatcherServerTLSSecretName = "imc-dispatcher-server-tls" //nolint:gosec // This is not a hardcoded credential
+	// IMCDispatcherTLSSecretName is the name of the tls secret for the imc dispatcher
+	IMCDispatcherTLSSecretName = "imc-dispatcher-tls" //nolint:gosec // This is not a hardcoded credential
```

Comment:
hmm, I'm not sure why we use 2 constants (it's probably a bug)

https://github.com/knative/eventing/blob/4541f4fe994e8a3ba914a2ce1489eed269521c04/config/channels/in-memory-channel-tls/imc-dispatcher-tls-certificate.yaml#L22

Reply:
@pierDipi should I switch to just using the one?